doc/reboot-windows.md

```diff
@@ -0,0 +1,35 @@
+# Reboot windows
+The CLUO `update-operator` can be configured to only reboot nodes during certain timeframes.
```

Comment:
add something like "Pre-reboot checks are prevented from running as well." since we check the window before applying the before-reboot label.